src/components/AddToMetamask/index.tsx

```diff
@@ -35,16 +39,26 @@ export default function AddTokenToMetaMask({ token, chainId }: { token: Token; c
             },
           },
         })
+      } catch (error) {
+        console.error(error)
+      }
+    } else {
+      if (!account) return
+      if (!provider) return
+      if (!isAddress(chainId, tokenAddress)) return
+      try {
+        await createUserAssociatedTokenAccount(provider, new PublicKey(tokenAddress), new PublicKey(account))
+      } catch (error) {
+        console.error(error)
       }
-    } catch (error) {
-      console.log(error)
     }
   }
-
+  if (!walletKey) return <></>
+  if (isEVM && walletKey === 'COINBASE') return <></> // Coinbase wallet no need to add since it automatically track token
```

Comment:
Let's help us bring it into constant: COINBASE, COIN98,  ...
<img width="422" alt="image" src="https://user-images.githubusercontent.com/33005392/201367721-0328914a-7b9d-42a1-82b3-8e05d431ff6d.png">

Reply:
`walletKey` is already `keyof typeof SUPPORTED_WALLETS`

Reply:
dont need constant I think